### DISCUSSION TITLE
Next.js app using Turborepo and Prisma shows error: PrismaClientInitializationError: Your schema.prisma could not be found.

### DISCUSSION THREAD

**Discussion by sebpowell** [vercel/turborepo#5610] [Help]
I have a project that uses Next.js and Prisma, which I setup as per the instructions here: https://turbo.build/repo/docs/handbook/tools/prisma

I've imported the PrismaClient into the app folder like so:

```TS
// apps/my-app/lib/prisma.ts 

import { PrismaClient } from "database";

const client = new PrismaClient();

export default client;
```

I then have an api route located inside the app directory that looks something like this:

```TS
// apps/my-app/app/api/route.ts

import prisma from "@/lib/prisma";

export async function POST(req: NextRequest) {
    const items = await prisma.items.findMany();
}
```

This works without any problems locally. However, once on Vercel, when I try calling that endpoint I get a 500 error that says the following:

```
PrismaClientInitializationError: Your schema.prisma could not be found, and we detected that you are using Next.js.
Find out why and learn how to fix this: https://pris.ly/d/schema-not-found-nextjs
```

Has anyone run into this before? Any help would be greatly appreciated.

**Comment by sebpowell** [comment]
Eventually found the solution – this is the issue, in case it's of use to anyone: https://www.prisma.io/docs/guides/other/troubleshooting-orm/help-articles/nextjs-prisma-client-monorepo

Following the steps described here solved the problem.

**Comment by sneht** [comment]
Try [This](https://github.com/sneht/turborepo-with-prisma) example it maybe helps
